### DISCUSSION TITLE
Induction factor optimization

### DISCUSSION THREAD

**Discussion by seltzera** [NatLabRockies/floris#835] [Q&A]
Is there a way to use FLORIS to optimize the induction factors in a wind farm? 

As far as I can tell, the induction factor is not a straightforward parameter adjustment. Since it is an implicit factor derived from the Cp and Ct curves relative to the wind speed, one would need to optimize the Cp and Ct values and then relate them to the induction factors. 

Is this correctly understood, or is there a direct way to optimize the induction factor with FLORIS?

**Comment by misi9170** [comment]
Hi @seltzera , that's correct, there's is currently not a way of controlling the axial induction factors. In FLORIS v4, which will be released in the next month or so but which you can [check out now if interested](https://github.com/NREL/floris/tree/v4), we have significantly refactored how [turbine operation is modeled](https://github.com/NREL/floris/pull/770) and, in particular, included the capability to [derate the turbines](https://github.com/NREL/floris/pull/783), that is, provide power setpoint lower than the available power. This has the effect of reducing the thrust coefficient and axial induction factor of the turbines. This still will not allow direct control of the axial induction factor, but the power setpoint could conceivably be calculated based on the desired axial induction before passing to FLORIS.
